### ISSUE TITLE
install man pages

### ISSUE THREAD

**Issue by bric3** [docker/for-mac#743]
Docker is super nice, but when not connected to internet it would be nice to have documentation in the standard manual pages e.g. `man`

```sh
~/opensource ○ ❯ docker -v
Docker version 1.12.2-rc1, build 45bed2c, experimental
~/opensource ○ ❯ man docker
No manual entry for docker
```

**Comment by yallop** [comment]
Thanks for the suggestion.  We plan to include and install the man pages in a future release.

**Comment by bric3** [comment]
You're welcome, that's small suggestion ;)
Thanks again for the work on docker*

**Comment by cmosetick** [comment]
@bric3 Here is something you can run to get the man pages while offline:
https://github.com/cmosetick/docker-manpages-osx

**Comment by bric3** [comment]
@cmosetick Thanks for the script ! It doesn't work here the md2man fails with this error. I don't know exactly what's wrong at this moment. I'll check that later when I have more free time ;)

```
docker: Error response from daemon: invalid header field value "oci runtime error: container_linux.go:247: starting container process caused \"exec: \\\"man/generate.sh\\\": stat man/generate.sh: no such file or directory\"\n".
```

**Comment by dunstontc** [comment]
Any update on this?

**Comment by bric3** [comment]
I understand docker for Mac team may have other priorities. In order to not distract you from more serious challenge/issues, how can we help in this?

**Comment by oxfist** [comment]
Yeah, how can we help in this? Using Docker almost on a daily basis and not having man pages directly on the terminal really hurts having a better, more accessible understanding of the tool and hence productivity.

**Comment by kyle-laracey** [comment]
Any news on this?

**Comment by kyle-laracey** [comment]
For anyone else who finds this, I just looked up the version of docker I have (via `docker -v`), then did something like

```bash
git clone https://github.com/docker/cli.git \
  && cd cli \
  && git fetch --tags \
  && git checkout $(docker version -f '{{.Client.Version}}') \
  && make -f docker.Makefile manpages
```

Then

```bash
for i in 1 5 8; do sudo cp man/man$i/* /usr/local/share/man/man$i; done
sudo rm /usr/local/share/man/whatis
sudo /usr/libexec/makewhatis  /usr/local/share/man
```

**Comment by bric3** [comment]
Or faster by passing the tag directly.

```shell
$ git clone --depth=1 --branch "v$(docker version -f '{{.Client.Version}}')" git@github.com:docker/cli.git docker-cli \
  && cd docker-cli \
  && make -f docker.Makefile manpages

$ export MANPATH="$MANPATH:$(pwd)/man"
$ man docker
```

To update the man pages 

```shell
$ cd docker-cli \
  && git checkout "v$(docker version -f '{{.Client.Version}}')"
  && make -f docker.Makefile manpages
```

By thanks to @kyle-laracey and @benknoble

**Comment by joaocfernandes** [comment]
Can docker-desktop mac please consider to install the man pages in the system? Thank you!!

**Comment by jasonmp85** [comment]
It's _five years_ since I last commented on this and why isn't it done.

**Comment by jdanbrown** [comment]
The [fish shell](https://github.com/fish-shell/fish-shell) (which is wonderful) generates its tab completion from manpages, so missing manpages in Docker.app means that fish users have no tab completion for the `docker` cli command.

Please revive this issue! (Now 8 years old.)
